pkg/resources/scim_integration.go

```diff
@@ -24,34 +27,45 @@ var scimIntegrationSchema = map[string]*schema.Schema{
 	},
 	"enabled": {
 		Type:        schema.TypeBool,
-		Required:    true,
+		Optional:    true,
 		Description: "Specify whether the security integration is enabled.",
+		DiffSuppressFunc: func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+			if d.GetRawConfig().AsValueMap()["enabled"].IsNull() {
+				return true
+			}
+			return oldValue == newValue
```

Comment:
Hmm, I think in this case the diff shouldn't be produced, but maybe I'm wrong.

Reply:
Fixed.